src/component/Adoption/ConfirmApplication.js

```diff
@@ -0,0 +1,7 @@
+import React from "react";
+
+const ConfirmApplicatio = () => {
```

Comment:
There is a typo here in the component name - missing 'n' at the end

Reply:
OK! Thanks!